.github/workflows/nodejs.yml

```diff
@@ -83,9 +81,6 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
-      - name: Install webpack ${{ matrix.webpack-version }}
-        run: npm i webpack@${{ matrix.webpack-version }}
-
```

Comment:
This is unnecessary and it's installing `schema-utils: "3.1.0"` which is breaking CI, probably in all loaders/plugins repo where we don't have schema-utils as a direct dependency.

Reply:
/cc @alexander-akait

Reply:
Just update lock file

Reply:
No luck here 😕  https://github.com/webpack-contrib/imports-loader/pull/104